PlasterTemplate/PlasterTemplates/GitLab/PlasterManifest.xml

```diff
@@ -16,12 +16,17 @@
     <parameter name="GitLabUserName" type="text" prompt="Module author's GitLab user account name" />
     <parameter name="GitLabEmail" type="text" prompt="Module author's GitLab email address" />
     <parameter name="GitLabURL" type="text" prompt="The URL of the GitLab server" />
+    <parameter name="GitLabHostname" type="text" prompt="The Hostname or IP of the GitLab server, without http(s):// or /" />
```

Comment:
Could you change `or /` to `or trailing /` please just to be super clear?

I also have some concerns over asking to omit `http(s)://` here and how that will affect other things but I see why you've done it. I'll have a think and see we can be a bit clever with this somehow.